challenges/prototypes.js

```diff
@@ -5,29 +5,62 @@
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
+function  CuboidMaker(props) {
+this.length = props.length;
+this.width = props.width;
+this.height = props.height
+}
+
 
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
   
   Formula for cuboid volume: length * width * height
 */
+function  CuboidMaker(props) {
+  this.length = props.length;
+  this.width = props.width;
+  this.height = props.height
+  
+}
 
-
+CuboidMaker.prototype.volume = function () {
+  return (this.length * this.width * this.height);
+};
 /* == Step 3: Surface Area Method ==
   Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
+function  CuboidMaker(props) {
```

Comment:
In prototypes, you have declared CuboidMaker 3 different times.  It doesn't give you any problems because function declarations aren't const, but it is unnecessary.  You only need to declare it once up top.  After the constructor is created, you can add to it's prototype anywhere afterwards.  It looks like you're declaring the function so that you can add to the prototype, but they're actually completely separate functions.  You can see as it's been closed by the `}`